tests/serial/time/explicit/parallel-coupling/DoManySmallSteps.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <precice-configuration>
-  <data:scalar name="DataOne" waveform-degree="0" />
-  <data:scalar name="DataTwo" waveform-degree="0" />
+  <data:scalar name="DataOne" />
+  <data:scalar name="DataTwo" />
```

Comment:
This is a slight modification of this test (and some similar ones). But I think it is fine because this test is not about interpolation and it simplifies the config.